nixos/modules/services/backup/postgresql-backup.nix

```diff
@@ -50,10 +50,10 @@ in {
         '';
       };
 
-      period = mkOption {
+      startAt = mkOption {
```

Comment:
Adding:

```
(mkRemovedOptionModule [ "services" "postgresqlBackup" "period" ] "A systemd timer is now used instead of cron. The starting time can be configured via services.postgresqlBackup.startAt")
```

`modules/rename.nix` would make sense.

Reply:
OK. Do we need to also mention the change in the release notes?

Reply:
Probably not, the error message generated from mkRemovedOptionModule should be self-explanatory.